src/main/java/com/twilio/sdk/resource/instance/Account.java

```diff
@@ -812,6 +812,31 @@ public UsageRecordList getUsageRecords() {
 	}
 
 	/**
+	 * Gets the Usage Record sub resource list with the given filters
+	 * <p/>
+	 * <a href="https://www.twilio.com/docs/api/rest/usage-records">https://www.twilio.com/docs/api/rest/usage-records</a>
+	 *
+	 * @param filters the filters
+	 * @return the usage records
+	 */
+	public UsageRecordList getUsageRecords(Map<String, String> filters, UsageRecordList.Type type) {
+		UsageRecordList list = new UsageRecordList(this.getClient(), filters, type);
+		list.setRequestAccountSid(this.getRequestAccountSid());
+		return list;
+	}
+
+	/**
+	 * Gets the Usage Record subresource list
+	 * <p/>
+	 * <a href="https://www.twilio.com/docs/api/rest/usage-records">https://www.twilio.com/docs/api/rest/usage-records</a>
+	 *
+	 * @return the usage records
+	 */
+	public UsageRecordList getUsageRecords(UsageRecordList.Type type) {
+		return this.getUsageRecords(new HashMap<String, String>(), type);
```

Comment:
I actually did this because this is how it is done nearly everywhere in the library currently. I can replace it with ```Collections.<String, String>emptyMap()``` if you'd like but we need to make sure it's immutable from that point to avoid any ```UnsupportedOperationException``` from the EMPTY_MAP implementation.